tailwind.config.js

```diff
@@ -13,6 +13,9 @@ module.exports = {
         grayy: "#e5e5e5",
         blueGray: "#5b6280",
       },
+      fontFamily: {
+        noto: ['"noto sans"'],
```

Comment:
Curious about this syntax of single quotes within double quotes?

Reply:
Read it in documentation, I think I can clean it up but it works